src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Security/LoginData.cs

```diff
@@ -2098,7 +2111,16 @@ public LoginPrototype(CDataContainer context, Login login)
             this.currentState   = new LoginPrototypeData(server, login);
             this.originalState  = (LoginPrototypeData) this.currentState.Clone();
             this.comparer       = new SqlCollationSensitiveStringComparer(server.Information.Collation);
-            this.securablePermissions = SecurableUtils.GetSecurablePermissions(this.exists, PrincipalType.Login, login, context);
+            try
+            {
+                this.securablePermissions = SecurableUtils.GetSecurablePermissions(this.exists, PrincipalType.Login, login, context);
+            }
+            catch (Exception ex)
+            {
+                Logger.Error($"Exception thrown while trying to get securables for login '{this.LoginName}'. Error message: '{ex.Message}'");
+                this.securablePermissions = new SecurablePermissions[0];
+                this.hidePermissions = true;
```

Comment:
Could you do the same here with the nullable? If this is just temporary code feel free to ignore though